### DISCUSSION TITLE
Spreading markers on a same location

### DISCUSSION THREAD

**Discussion by RicardoBrito1938** [uber/react-map-gl#1519] [Q&A]
Hello guys, my cluster is working fine, but i have some cases where two markers have the same exact location, i would like to know if there is any solution fo spread these markers, something similar to this :

![image](https://user-images.githubusercontent.com/51454097/125124325-47f42700-e0ce-11eb-9f7f-40fd50c85ed1.png)

**Comment by louislefebvre** [comment]
Hey :) I am facing the same situation, did you find a solution or an alternative? Thank you

**Reply by RicardoBrito1938** [reply]
Hello, in that time, the solution I used, I made another cluster, and when I clicked, I changed slightly the location in a way they were almost in same place, but different enough to see there are two or more of them
